src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -28,6 +28,10 @@
     <DefineConstants>$(DefineConstants);OFFICIAL_BUILD</DefineConstants>
   </PropertyGroup>
   <ItemGroup Label="Project References">
+    <ProjectReference Include="..\..\Compilers\Core\MSBuildTask\MSBuildTask.csproj">
+      <Project>{d874349c-8bb3-4bdc-8535-2d52ccca1198}</Project>
+      <Name>MSBuildTask</Name>
```

Comment:
I would recommend adding the `<IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>` line you see in the other ProjectReferences. Without it the VSIX tooling will default to including the built output + some amount of dependencies, where "some amount" is unclear. For all you know it might copy in half of MSBuild.